app/helpers/sufia_helper.rb

```diff
@@ -2,4 +2,17 @@ module SufiaHelper
   include ::BlacklightHelper
   include Sufia::BlacklightOverride
   include Sufia::SufiaHelperBehavior
+
+  def render_visibility_label(document)
```

Comment:
Assignment Branch Condition size for render_visibility_label is too high. [15.3/15]